src/LiberisLabs.CompaniesHouse/Response/CompanyFiling/FilingHistoryItem.cs

```diff
@@ -29,6 +30,9 @@ public class FilingHistoryItem
         [JsonProperty(PropertyName = "description")]
         public string Description { get; set; }
 
+        [JsonProperty(PropertyName = "description_values")]
+        public Dictionary<string, dynamic> DescriptionValues { get; set; }
```

Comment:
We could add a method GetDescription(string format) where format would be the value from the "api-enumerations" and it would use a private dynamic object to produce end result? That would keep description_values private and not visible to the consumer.